notebooks/kubeflow_pipelines/pipelines/solutions/kfp_pipeline_vertex_automl_batch_preds.ipynb

```diff
@@ -0,0 +1,569 @@
+{
```

Comment:
<p>I'd give more explanation here on where the pipeline batch prediction output will be stored. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>